js/app.js

```diff
@@ -1,2 +1,88 @@
 $(function () {
+// pseudocode:
+// 1) Create input box to input city name -
+// 2) Create clickable submit button to input a city -
+// 3) create function to call the Openweather API to retrieve results for the city entered by the user
+// 4) Create a field to display results
+//      Results should include:
+//      - [ ] City name
+//      - [ ] Current temperature (displayed in Fahrenheit)
+//      - [ ] Weather description
+//      - [ ] Min temp
+//      - [ ] Max temp
+// 5) create a function to display results to display field
+// create an if statement to determine color of temperature display depending on temperature
+//
+$('#submit-btn').click(() => {
+  event.preventDefault()
+  //console.log('Search button is working')
+
+  searchTerm = $('#city-value').val()
+  console.log(searchTerm)
+
+  searchCity(searchTerm)
+
+  })
+
+  function searchCity(searchTerm) {
+    const url = `https://api.openweathermap.org/data/2.5/weather?q=${searchTerm}&APPID=6c020773c72c398344ed5f2a19411549`
+
+  // make API request using #.ajax()
+      $.ajax({
+        url: url,
+        type: 'GET'
+
+      })
+      .done((response) => {
+        console.log(response)
+        // get response from successful API call and
+        // pass response data to the updateUi() function
+        displayResults(response)
+
+      })
+      .fail((error) => {
+        console.log(error)
+        alert('an error occurred')
+      })
+    }
+
+    function displayResults(city) {
+      // extract the information from the objects
+      const cityName = city.name
+
+      const cityTempK = city.main.temp
+      const cityTempF = ((((parseInt(cityTempK)) - 273.15) * 9) / 5) + 32
```

Comment:
@anniecdavis27 nice work here doing the conversion, however OpenWeatherMap's API does this work for you by passing an additional request parameter with the API call:  `units: 'imperial'` - will return the temperature in Fahrenheit

See the docs here: https://openweathermap.org/current#data 

And here's how I use it in the solution: https://github.com/jsd20191008/weather-api-app-solution/blob/master/js/main-jquery-ajax.js#L50